client/components/payments-activity/payments-data-tile.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { formatCurrency } from 'wcpay/utils/currency';
+import Loadable from '../loadable';
+interface PaymentsDataTileProps {
+	id: string; // id of the element can be used for CSS styling
+	label: string; // The label appears as title of the tile
+	currencyCode: string; // The currency to be displayed in the tile
+	tooltip?: React.ReactElement; // For optionally passing the ClickTooltip component
+	amount?: number; // The amount to be displayed in the tile
+	isLoading?: boolean; // Loading state of the tile
+	reportLink?: string; // Optional hover link to view report
+}
+
+const PaymentsDataTile: React.FC< PaymentsDataTileProps > = ( {
+	id,
+	label,
+	currencyCode,
+	tooltip,
+	amount = 0,
+	isLoading = false,
+	reportLink,
+} ) => {
+	return (
+		<div id={ id } className="wcpay-payments-data-highlights__item">
+			<p className="wcpay-payments-data-highlights__item__label">
+				<span>{ label }</span>
+				{ ! isLoading && tooltip }
+			</p>
+			<div className="wcpay-payments-data-highlights__item__wrapper">
+				<p
+					className="wcpay-payments-data-highlights__item__wrapper__amount"
+					aria-labelledby={ id }
+				>
+					<Loadable
+						isLoading={ isLoading }
+						display="inline"
+						placeholder="loading amount"
+						value={ formatCurrency( amount, currencyCode ) }
+					/>
+				</p>
+				{ reportLink && (
+					<a
+						href={ reportLink }
+						className="wcpay-payments-data-highlights__item__wrapper__report-link"
+					>
+						{ __( 'View report', 'woocommerce_payments' ) }
+					</a>
```

Comment:
💭 I recommend using `@woocommerce/components` `<Link />` here for faster client-side navigation.

```suggestion
					<Link href={ reportLink }>
						{ __( 'View report', 'woocommerce_payments' ) }
					</Link>
```

We may have to change the custom CSS to target `a` rather than this classname, since `<Link />` doesn't seem to accept a `className`. 🫢

Reply:
Does the `Link` have an (inbuilt) class (or similar) we could target with an ancestor selector?

i.e. we add a class to our outer container and anchor all the CSS tweaks to that (hopefully we don't need lots of CSS tweaks)

Reply:
> Does the Link have an (inbuilt) class (or similar) we could target with an ancestor selector?

✅ Ah, I missed that it does actually pass though props. We're manually declaring types for `@woocommerce/components` and we can simply add `className` here:

https://github.com/Automattic/woocommerce-payments/blob/a0299a184a346a3e0d4a9b84a8f97416c987f470/client/deposits/declarations.d.ts/#L31-L39


> hopefully we don't need lots of CSS tweaks

It looks like we're removing the underline and show/hiding based on the design.

Reply:
> It looks like we're removing the underline and show/hiding based on the design.

Whenever the design deviates from our components we should aim to either harmonise design with working component or adapt component to match design (rather than override/customise one-off). I don't think designers intend each UI to have custom look/feel/behaviour.

Reply:
Thanks for the great points. I will keep in mind to use the Link component that includes our design . Implemented here d53c1de